capellambse/_diagram_cache.py

```diff
@@ -219,6 +220,255 @@ def _sanitize_filename(fname: str) -> str:
     return fname
 
 
+def circle_extents(
+    element: etree.Element,
```

Comment:
Thank you!
I analyzed that and got to the following:
One must have `types-lxml` installed to get `mypy` errors for `etree.Element` we see in the GitHub pre-commit job.

`types-lxml` is missing in the `pyproject.toml`. IMHO it must be listed as optional `dev` dependency when it is configured in `pre-commit` so that the linter will already complain during decelopment.

So we may want to add:

```toml
[project.optional-dependencies] 
dev = ["types-lxml"]
```
to the `pyproject.toml`

I replaced all `etree.Element` by `etree._Element`.